### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by abhisheksunil2201**
Added homepage

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -34,14 +37,50 @@ a {
...
+  padding: 1rem;
+  border: none;
+  border-radius: 10px;
+  font-weight: 700;
+  font-size: 1.2rem;
+  cursor: pointer;
+  margin-top: 1rem;
+}
+.get-started-btn.github {
+  background-color: rgba(255, 255, 255, 0.1);
```
Comment (ravikumar1002) [inline style.css:70]: use color variable

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

**Review by KamiyaGaikwad — APPROVED** [review]
Looks good to merge

**Comment by inductionotg** [comment]
LOOKS good to merge

**Pull request merged**